doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -377,34 +387,34 @@ EOF
 Check your Gateway policies are enforced as follows:
 
 ```bash
-kubectl get dnspolicy ${gatewayName}-dnspolicy -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
-kubectl get authpolicy ${gatewayName}-auth -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
-kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get dnspolicy ${GATEWAY_NAME}-dnspolicy -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get authpolicy ${GATEWAY_NAME}-auth -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
+kubectl get ratelimitpolicy ${GATEWAY_NAME}-rlp -n ${GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="Enforced")].message}'
 ```
 
 Check your listener is ready as follows:
 
 ```
-kubectl get gateway ${gatewayName} -n ${gatewayNS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
+kubectl get gateway ${GATEWAY_NAME} -n ${GATEWAY_NS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
 ```
 
 ### Step 6 - Test connectivity and deny all auth 
 
 You can use `curl` to hit your endpoint. You should see a `403`. Because this example uses Let's Encrypt staging, you can pass the `-k` flag:
 
 ```bash
-curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
+curl -k -w "%{http_code}" "https://$(kubectl get httproute test -n ${GATEWAY_NS} -o=jsonpath='{.spec.hostnames[0]}')"
```

Comment:
I think it would be pretty handy to call out that you can execute this in a curl container if you are on a mac

```
echo "https://$(kubectl get httproute test -n ${GATEWAY_NS} -o=jsonpath='{.spec.hostnames[0]}')"
kubectl run mycurlpod --image=curlimages/curl -i --tty -- sh
export HOST={output from echo}
curl -k -w "%{http_code}" "https://${HOST}"

Reply:
My mac is not able to connect to the network of the kind cluster at the  moment so this is a way to do it without having to debug all that

Reply:
this is meant to be on openshift so they wouldn't be using kind